py-polars/polars/lazy/expr.py

```diff
@@ -1738,6 +1739,138 @@ def str_concat(self, delimiter: str = "-") -> "Expr":  # type: ignore
         """
         return wrap_expr(self._pyexpr.str_concat(delimiter))
 
+    def sin(self) -> "Expr":
+        """
+        Compute the element-wise value for Trigonometric sine on an array
+
+        Returns
+        -------
+        Series of dtype Float64
+
+        Examples
+        >>> df = pl.DataFrame({"a": [0.0]})
+        >>> df.select(pl.col("a").sin())
+        shape: (1, 1)
+        ┌────────────────────┐
+        │ a                  │
+        │ ---                │
+        │ f64                │
+        ╞════════════════════╡
+        │ 0.8414709848078965 │
+        └────────────────────┘
+        """
+        return wrap_expr(np.sin(self))  # type: ignore
```

Comment:
I don't believe we need to `wrap_expr` here. As we should already get an `Expr` returned from the `__ufunc__` implementation.